install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -390,7 +390,7 @@ each PV.
 
 They are many ways that you can use scripts to automate the above tasks. You can
 use an
-link:https://github.com/openshift/openshift-ansible/tree/master/roles/kube_nfs_volumes[example
+link:https://github.com/openshift/openshift-ansible/tree/4c2a6db2b968abdf7ab641340700856ff179e8e3/roles/kube_nfs_volumes[example
```

Comment:
Looks like this was removed from examples. Can you please use replace the link with `https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_node_certificates` instead?